src/database/schema/swaps_v2.ts

```diff
@@ -0,0 +1,56 @@
+import { relations } from 'drizzle-orm';
+import { pgTable, serial, varchar, integer, timestamp, boolean, unique, jsonb, numeric } from 'drizzle-orm/pg-core';
+
+import { chains } from './chains';
+import { tokens } from './tokens';
+
+// Define the swaps table without foreign key constraints on poolId
+export const swapsTableV2 = pgTable(
+  'swaps_v2',
+  {
+    id: serial('id').primaryKey(),
+    chainId: integer('chain_id')
+      .notNull()
+      .references(() => chains.id, { onDelete: 'cascade' }),
+    transactionHash: varchar('transaction_hash', { length: 256 }).notNull(),
+    user: varchar('user', { length: 256 }).notNull(),
```

Comment:
address is always 42 characters
```suggestion
    user: char('user', { length: 42 }).notNull(),
```